application/src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -60,6 +68,23 @@ public DirectTransferGenerator(
     this.issueStore = issueStore;
     this.radiusByDuration = radiusByDuration;
     this.transferRequests = transferRequests;
+    this.carsAllowedStopMaxTransferDurationsForMode = DurationForEnum.of(StreetMode.class).build();
```

Comment:
Maybe we should always include this with some default value that is the same as the normal transfer duration limit.

Reply:
This has changed quite a lot, but you can look at how I handle an empty parameter